connector/connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectProtoSuite.scala

```diff
@@ -173,6 +175,15 @@ class SparkConnectProtoSuite extends PlanTest with SparkConnectPlanTest {
     comparePlans(connectPlan2, sparkPlan2)
   }
 
+  test("Test Range") {
+    comparePlansDatasetLong(connect.range(None, 10, None, None), spark.range(10))
```

Comment:
how about we call `spark.range(10).toDF` then we don't need to add `comparePlansDatasetLong`?

Reply:
Let me try to see if it gives an exact plan.


Another idea might be we just compare the result through `collect()` so we do not compare the plan on this case.

Reply:
Oh the `.toDF()` just convert things into DataFrame. 

It has removed the `comparePlansDatasetLong`